frontend/src/scenes/experiments/ExperimentView/components.tsx

```diff
@@ -189,8 +189,22 @@ export function NoResultsEmptyState(): JSX.Element {
 
     // Validation errors return 400 and are rendered as a checklist
     if (experimentResultCalculationError?.statusCode === 400) {
+        let parsedDetail
```

Comment:
small nit, for easier readability and future-proofing if say, later on we stop relying on the early return in catch
```suggestion
        let parsedDetail = {}
```